siac/hostcmd.go

```diff
@@ -51,6 +52,61 @@ Doing so will override the standard connectivity checks.`,
 	}
 )
 
+// parseSize converts strings of form 10GB to a size in bytes
+// If parsing fails the original value is returned unmodified
+func parseSize(value string) string {
```

Comment:
Agreed. I will change that. (I just pulled value since that was the variable name in the original function but doesn't make much sense out of context)